nemo/collections/nlp/modules/common/transformer/transformer_decoders.py

```diff
@@ -124,8 +124,16 @@ def __init__(
         ffn_dropout: float = 0.0,
         hidden_act: str = "relu",
         pre_ln: bool = False,
+        pre_ln_final_layer_norm: bool = True,
     ):
         super().__init__()
+        self.pre_ln = pre_ln
```

Comment:
do you need to store these values? you can check the final_layer_norm in the forward to see if it None or not.